### DISCUSSION TITLE
What do the dBm values in the console refer to?

### DISCUSSION THREAD

**Discussion by peff74** [tbnobody/OpenDTU#1031] [Q&A] [answered]
I have recently had problems with disconnections from the inverter.
Too much foliage on the bushes^^
After shielding the nrf it worked again.

However, I have now noticed dBm value in the console, which I had not seen before?
![2023-06-13 08_40_54-OpenDTU](https://github.com/tbnobody/OpenDTU/assets/91614910/3a0f3222-6ddf-4061-99b3-4bf74e861168)

But no matter what I do (increase power or go closer to the inverter) they always stay the same

**Comment by tbnobody** [comment]
Please see https://github.com/tbnobody/OpenDTU/issues/980#issuecomment-1568983333

not sure if it is reliable

**Reply by peff74** [reply]
O.K. I only have nf24 in use no CMT2300A.
that's why -80 is displayed for all of them, am I right?

**Reply by tbnobody** [reply, marked as answer]
No, the image mentioned above shows a method for the nrf which just distincts between >-64db und <-64db... i replace this by -80db and -30db 
https://github.com/tbnobody/OpenDTU/blob/3562e85b5ec110c73fba0b72858e47bf9b667ea6/lib/Hoymiles/src/HoymilesRadio_NRF.cpp#L58

And to be honest, I see most of the time the -80db... one or two times a day I see -30db but never the less, my connection works very reliable... therefor I am not 100% sure how good this indicator really is. currently I included it to gather some experience

**Reply by peff74** [reply]
Ok I have probably not read in detail.
The question is clarified. Thanks
